scripts/plugins/support-pssl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Open/Close port 6640 when pssl is used in ovs
```

Comment:
Do we need to hard code 6640 ? Can it be passed as a parameter or at least define a variable